### PR TITLE
NIAD-1296: GitHub bugs

### PR DESCRIPTION
**Pull request by bartek-sarul**
## Description

Please explain the changes you have made here.

## Jira Ticket

Please add a link to the related Jira ticket (if any).

## Checklist

These are items (excluding GitHub Checks) which should be confirmed before a branch is ready to merge.

- [ ] Acceptance Criteria met
- [ ] Commit messages are meaningful
- [ ] Manually tested
- [ ] Self-reviewed your code
- [ ] Code reviewed from two other developers
- [ ] If your pull request depends on any other, please link them in the description
- [ ] main branch is passing/green on CI
- [ ] Add/update any relevant documentation

### CODE REVIEW

File: src/main/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/PatientMapper.java [hunk 1]
```diff
@@ -71,10 +73,16 @@ private void mapAddress(final UnstructuredAddress unstructuredAddress, final Pat
...
+        patient.addIdentifier()
+            .setSystem(NHS_NUMBER_SYSTEM)
+            .setValue(nhsNumber)
+            .addExtension()
+            .setUrl("https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-NHSNumberVerificationStatus-1")
+            .setValue(new CodeableConcept().addCoding(new Coding()
+                .setSystem("https://fhir.hl7.org.uk/STU3/CodeSystem/CareConnect-NHSNumberVerificationStatus-1")
+                .setCode("01") //TODO awaiting NHS IOPS to provide answer
+                .setDisplay("Number present and verified")
+            ));
```
Comment (matthewdoreilly) [inline src/main/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/PatientMapper.java:85]: Formatting is a bit strange to me. I'd format it like this:
```
.setValue(new CodeableConcept().addCoding(
        new Coding()
            .setSystem("https://fhir.hl7.org.uk/STU3/CodeSystem/CareConnect-NHSNumberVerificationStatus-1")
            .setCode("01") //TODO awaiting NHS IOPS to provide answer
            .setDisplay("Number present and verified")
));
```

Also is getting the value of the code here from the IOPS team still in progress?

File: src/main/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/PatientMapper.java [hunk 2]
```diff
@@ -78,10 +78,11 @@ private void mapNhsNumberIdentifier(final String nhsNumber, final Patient patien
             .setValue(nhsNumber)
             .addExtension()
             .setUrl("https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-NHSNumberVerificationStatus-1")
-            .setValue(new CodeableConcept().addCoding(new Coding()
-                .setSystem("https://fhir.hl7.org.uk/STU3/CodeSystem/CareConnect-NHSNumberVerificationStatus-1")
-                .setCode("01") //TODO awaiting NHS IOPS to provide answer
-                .setDisplay("Number present and verified")
+            .setValue(new CodeableConcept().addCoding(
```
Comment (matthewdoreilly) [inline src/main/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/PatientMapper.java:81]: Just noticed above we set the value to the `nhsNumber`, and then set it again here. I assume the first `setValue` isn't needed if we're overwriting it?

Comment (matthewdoreilly) [reply src/main/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/PatientMapper.java:81]: Nevermind, just noticed this is an extension

**Review by matthewdoreilly — APPROVED** [review]
Looks good 👍

**Review by mattd-kainos — APPROVED** [review]

**Pull request merged**